lib/api/objectGetTagging.js

```diff
@@ -0,0 +1,88 @@
+import async from 'async';
+import { errors } from 'arsenal';
+
+import { decodeVersionId, getVersionIdResHeader }
+  from './apiUtils/object/versioning';
+
+import { metadataValidateBucketAndObj } from '../metadata/metadataUtils';
+import { pushMetric } from '../utapi/utilities';
+import collectCorsHeaders from '../utilities/collectCorsHeaders';
+import { convertToXml } from './apiUtils/object/tagging';
+
+/**
+ * Object Get Tagging - Return tag for object
+ * @param {AuthInfo} authInfo - Instance of AuthInfo class with requester's info
+ * @param {object} request - http request object
+ * @param {object} log - Werelogs logger
+ * @param {function} callback - callback to server
+ * @return {undefined}
+ */
+export default function objectGetTagging(authInfo, request, log, callback) {
+    log.debug('processing request', { method: 'objectGetTagging' });
+
+    const bucketName = request.bucketName;
+    const objectKey = request.objectKey;
+
+    const decodedVidResult = decodeVersionId(request.query);
+    if (decodedVidResult instanceof Error) {
+        log.trace('invalid versionId query', {
+            versionId: request.query.versionId,
+            error: decodedVidResult,
+        });
+        return process.nextTick(() => callback(decodedVidResult));
+    }
+    const reqVersionId = decodedVidResult;
+
+    const metadataValParams = {
+        authInfo,
+        bucketName,
+        objectKey,
+        requestType: 'bucketOwnerAction',
```

Comment:
why bucketOwner?  I would think that you would need object get acl rights? did we do bucketOwnerAction on put?  If so, that probably needs to be updated.